### ISSUE TITLE
Sundanese dialects naming

### ISSUE THREAD

**Issue by Maramukiri** [glottolog/glottolog#973]
I think the naming of dialects in Sundanese is a bit strange. Should, follow the name of the dialect in another language does not have to be followed by the name of the language in the dialect. As an example, "Tangerang Sundanese", there it only has to be written "Tangerang" without having to add the word "Sundanese", because people already know that this dialect is included in Sundanese.

**Comment by d97hah** [comment]
There's also the Malay/Indonesian spoken in Tangerang

Pada tanggal Sel, 11 Jul 2023 pukul 21.43 Maramukiri <
***@***.***> menulis:

**Comment by Maramukiri** [comment]
Yes, maybe it can be slightly distinguished. But what about the others? Like Binong Sundanese and Pandeglang Sundanese? Isn't there only Sundanese dialect there?
